DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: linl
 Type: Package
 Title: 'linl' is not 'Letter'
-Version: 0.0.2
+Version: 0.0.3
```

Comment:
I tend to call these 0.0.2.1 before making it final for CRAN but details...

Reply:
Done in https://github.com/eddelbuettel/linl/pull/18/commits/a833f4abdb17c85064fadb8e6344855aca039c43